service/telemetry/internal/otelinit/config.go

```diff
@@ -224,6 +225,14 @@ func initOTLPgRPCExporter(ctx context.Context, otlpConfig *config.OTLPMetric) (s
 		opts = append(opts, otlpmetricgrpc.WithEndpoint(u.Host))
 		if u.Scheme == "http" {
 			opts = append(opts, otlpmetricgrpc.WithInsecure())
+		} else {
+			if otlpConfig.Certificate != nil {
```

Comment:
Nit, but do we shouldn't need that extra indentation here

```suggestion
		} else if otlpConfig.Certificate != nil {
```

Reply:
Ah yes, I was thinking about future code there and also didn't realize I should run make lint (I'm new here)... fixed.